consoleme/celery_tasks/celery_tasks.py

```diff
@@ -573,8 +573,26 @@ def cache_policies_table_details() -> bool:
     if all_s3_errors:
         s3_errors = json.loads(all_s3_errors)
 
+    allowed_tags = config.get("roles.allowed_tags", {})
+    allowed_arns = config.get("roles.allowed_arns", [])
     for arn, role_details_j in all_iam_roles.items():
         role_details = ujson.loads(role_details_j)
+        role_details_policy = ujson.loads(role_details.get("policy"), {})
```

Comment:
Ahhh, I don't seem to have permission to push to this branch... this line should be:
`ujson.loads(role_details.get("policy", {}))` instead of `ujson.loads(role_details.get("policy"), {})`
and that should fix the unit tests.

Reply:
oof, thanks!